verilog/tools/syntax/export_json_examples/verible_verilog_syntax_test.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python3
+# Copyright 2017-2020 The Verible Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""VeribleVerilogSyntax test"""
+
+
+import unittest
+import verible_verilog_syntax
```

Comment:
I am not entirely sure how bazel organizes the import hierarchy, but I'd expected it to be in this particular directory. So even if local import works by accident, we should use the fully qualified name (similar how we deal with includes in C++).

So I imagine something like this ?:
```
from verilog.tools.syntax.export_json_examples import verible_verilog_syntax
```

Reply:
I've added `imports = ["."],` to `verible_verilog_syntax_py` build rules (it appends the path to the `PYTHONPATH`). Is this OK?

While your solution would work for the test, it could cause some confusion in `print_modules.py` and `print_tree.py` (which had the same problem). These examples should work without changes after being copied (together with `verible_verilog_syntax.py`) out of Verible source base.

Related commit: https://github.com/google/verible/pull/616/commits/feb21bf4d98ee9e34844686c49b55c0f7fb68235